### PR TITLE
Update Language Feature Status.md with VB information

### PR DESCRIPTION
**Pull request by AlekseyTs**

### CODE REVIEW

**Comment by AlekseyTs** [comment]
@jcouv Please review

File: docs/Language Feature Status.md [hunk 1]
```diff
@@ -16,6 +16,13 @@ efforts behind them.
 | [`field` keyword in properties](https://github.com/dotnet/csharplang/issues/140) | [field-keyword](https://github.com/dotnet/roslyn/tree/features/field-keyword) | [Merged into 17.12p3](https://github.com/dotnet/roslyn/issues/57012) | [Youssef1313](https://github.com/Youssef1313), [cston](https://github.com/cston) | [333fred](https://github.com/333fred), [RikkiGibson](https://github.com/RikkiGibson) | [CyrusNajmabadi](https://github.com/CyrusNajmabadi) | [CyrusNajmabadi](https://github.com/CyrusNajmabadi) |
 | [First-class Span Types](https://github.com/dotnet/csharplang/issues/7905) | [FirstClassSpan](https://github.com/dotnet/roslyn/tree/features/FirstClassSpan) | [Merged into 17.13p1](https://github.com/dotnet/roslyn/issues/73445) | [jjonescz](https://github.com/jjonescz) | [cston](https://github.com/cston), [333fred](https://github.com/333fred) | | [333fred](https://github.com/333fred), [stephentoub](https://github.com/stephentoub) |
 
+# Working Set VB
```
Comment (jcouv) [inline docs/Language Feature Status.md:19]: This could be VB 17.13, since we've already merged it and added the language version.

Comment (paul1956) [reply docs/Language Feature Status.md:19]: I can't seem to find feature list of VB versions between 15.x and the current latest before this PR.

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 1)

**Pull request merged**